obj.js

```diff
@@ -334,6 +372,15 @@ export default class BaseProtocol {
     obj[payload.name] = value;
   }
 
+  async observeListener(objDescrJSON) {
+    let obj = this.getRemoteObject(objDescrJSON.idLocal);
```

Comment:
```suggestion
    /* We get the first notification for the object when we subscribe to it;
      this happens when we first encounter the object on the server.
      At this point the remote stub does not yet exist on the client,
      since we have not yet sent the object description to the client.
      The subscription callback doesn't know that because the object already
      has a local id by this point, since we need that id in order to establish
      the subscription without creating a reference loop. */
    let obj = this.getRemoteObject(objDescrJSON.idLocal);
```